### PR TITLE
[TfL] Mark reports as safety critical & highlight as such in UI

### PR DESCRIPTION
**Pull request by davea**
Connects https://github.com/mysociety/fixmystreet-commercial/issues/1603

### CODE REVIEW

File: bin/tfl/import_categories [hunk 1]
```diff
@@ -89,6 +89,24 @@ for my $group (keys %$groups) {
...
+                {
+                    name => "Yes",
+                    key => "yes"
+                },
+                {
+                    name => "No",
+                    key => "no"
+                }
+            ]
+        });
```
[the remark this answers is not in the document]
Comment (davea) [reply bin/tfl/import_categories]: Yes, the previous approach wasn't quite right. Now all reports have a the safety critical yes/no field regardless of category.

File: templates/web/base/report/_item.html [hunk 2]
```diff
@@ -28,6 +28,11 @@
     [% END %]
 [% END %]
 
+[%~ TRY ~%]
+  [% item_extra_class = PROCESS 'report/_item_extra_class.html' %]
```
[the remark this answers is not in the document]
Comment (davea) [reply templates/web/base/report/_item.html]: Nice, will try that ta.

**Comment by davea** [comment]
> what if they change the answer to the safety critical question?

Who do you mean? Inspection staff? As it's a hidden field for users.

**Comment by dracos** [comment]
Yes, if an inspector changes the safety critical answer.

File: web/cobrands/tfl/base.scss [hunk 3]
```diff
@@ -277,4 +277,13 @@ ol.big-numbers>li {
     margin: 0
 }
 
+/* Safety critical reports */
+.item-list__item--safety-critical {
+    background-color: #ffb6b6;
```
Comment (dracos) [inline web/cobrands/tfl/base.scss]: Sorry not to think about this sooner, but we must not rely only on colour to show something like this: https://www.w3.org/TR/WCAG21/#use-of-color
Can it also prefix the title with `Safety Critical` or `[SC]` or have some other non-colour way of showing it is such a report.

**Pull request merged**